marklogic-data-hub-central/ui/src/components/common/hc-tag/hc-tag.tsx

```diff
@@ -0,0 +1,36 @@
+import React from "react";
+import styles from "./hc-tag.module.scss";
+import iconCloseTag from "../../../assets/icon_closeTag.svg";
+interface Props {
+  label: string;
+  ariaLabel?: string;
+  style?: React.CSSProperties;
+  className?: string;
+  dashed?: boolean;
+  color?: "black" | "blue" | "green" | "grey" | "magenta" | "red" | "yellow";
+  closable?: boolean;
+  visible?: boolean;
+  onClose?: () => void;
+}
+
+const colors = {
+  "black": styles.spanTagBlack, "blue": styles.spanTagBlue, "green": styles.spanTagGreen, "grey": styles.spanTagGrey,
+  "magenta": styles.spanTagMagenta, "red": styles.spanTagRed, "yellow": styles.spanTagYellow
+};
+
+const HCTag: React.FC<Props> = (props) => {
+  return (
+    <span aria-label={props.ariaLabel}
```

Comment:
In a long list of tags, the label and close icon can break in the middle at a new line. Can we use CSS to avoid that break?
<img width="905" alt="tag" src="https://user-images.githubusercontent.com/477757/132918175-8e142571-305f-413e-8617-21cdcd80e1c6.png">

Reply:
Nice catch! I tested with two lines but this didn't happen to me

Reply:
Thank you Mike. Done!
![Capture](https://user-images.githubusercontent.com/89084298/133093546-6511546c-efcb-4157-b0d3-35b3aeff906b.PNG)